### PR TITLE
Consolidate test workflows and point to main branch

### LINKED ISSUE #888: Consolidate test.yml and sveltekit.yml workflows

### PR DESCRIPTION
**Pull request by eyeseast**
Please check that I didn't miss any tests.

Closes #888

### CODE REVIEW

File: public/mockServiceWorker.js [hunk 1]
```diff
@@ -2,13 +2,15 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (1.3.5).
+ * Mock Service Worker.
  * @see https://github.com/mswjs/msw
  * - Please do NOT modify this file.
  * - Please do NOT serve this file on production.
```
Comment (allanlasser) [inline public/mockServiceWorker.js:8]: Are we sure we're not serving this file in production?

**Review by allanlasser — COMMENTED** [review]
Chromatic is missing from this refactor, I believe it's still configured to look for pulls on the `sveltekit` branch.

**Comment by allanlasser** [comment]
We've also got some specific includes in `vite.config:test` that can be relaxed:

```
include: [
  "src/lib/**/*.{test,spec}.{js,ts}",
  "src/routes/**/*.{test,spec}.{js,ts}",
  "src/lib/utils/tests/pageSize.test.ts",
],
```

to 

```
include: [
  "src/**/*.{test,spec}.{js,ts}",
],
```

File: public/mockServiceWorker.js (hunk 1 above)
Comment (eyeseast) [reply public/mockServiceWorker.js:8]: Not on this branch https://next.www.documentcloud.org/mockServiceWorker.js/

But some _other_ branches https://legacy.www.documentcloud.org/mockServiceWorker.js

**Review by allanlasser — APPROVED** [review]

**Pull request merged**